src/components/Collection/Series/EditSeriesModal.tsx

```diff
@@ -48,18 +48,17 @@ const EditSeriesModal = () => {
   const dispatch = useDispatch();
 
   const onClose = useEventCallback(() => {
-    dispatch(setStatus(false));
+    dispatch(setSeriesId(-1));
   });
 
-  const show = useSelector((state: RootState) => state.modals.editSeries.status);
   const seriesId = useSelector((state: RootState) => state.modals.editSeries.seriesId);
 
   const [activeTab, setActiveTab] = useState('name');
 
   return (seriesId === -1)
     ? null
     : (
-      <ModalPanel show={show} onRequestClose={onClose} header="Edit Series" size="md" noPadding noGap>
+      <ModalPanel show={seriesId !== -1} onRequestClose={onClose} header="Edit Series" size="md" noPadding noGap>
         <div className="flex h-[26rem] flex-row gap-x-6 p-6">
           <div className="flex shrink-0 gap-y-6 font-semibold">
```

Comment:
Remove the `(seriesId === -1) ? null :` part otherwise modal transition won't work

Reply:
Went mad trying to work out which transition doesn't work as it looked like it was... Took a dozen mini-recordings for me to realise it was the exit transition 🙃

Fixed in the latest commit 👍